### DISCUSSION TITLE
ART GENERATING WITH BLANK WHITE BACKGROUND

### DISCUSSION THREAD

**Discussion by meltapollo** [nftchef/art-engine#273] [Help + Q&A]
Hello, when i run my generations all the images get generated with no background. its just blank white. When i look at the DNA it is showing there is 2 background attributes assigned.

They are also generating only using 1 body trait. Idk whats going on but the only traits that are changing are the eyes and hats.

All my images have a rarity # at the end of them and my layers order is correct. Appreciate any help.

const background = {
  generate: true,
  brightness: "80%",
};

const layerConfigurations = [
  {
    growEditionSizeTo: 10,
    namePrefix: "Series 2", // Use to add a name to Metadata `name:`
    layersOrder: [
      { name: "Background" },
      { name: "Base"},      
      { name: "Body" },
      { name: "Mouth" },
      { name: "Eyes" },
      { name: "Mask-hat" },
      { name: "Accessory" },
    ],
  },

HERE IS THE DNA: THE BACKGROUNDS HAVE #100 AT THE END. IT SHOWS 2 BUT THERE SHOULD ONLY BE 1.. 

DNA: [
  '0.13:Orange#100.png',
  '1.7:Pink Powder#100.png',
  '2.18:Striped Shirt#50.png',
  '3.4:Buck Teeth#70.png',
  '4.18:Original eyes .gif',
  '4.25:VR Headset#30.png',
  '5.17:Purple Durag#60.png',
  '6.9:Parrot#100.png'
]

**Comment by nftchef** [comment]
This sounds like a naming/organizing issue with the image files and we would need to see how they are set up so figure out the issue.
